pkg/danm/danm.go

```diff
@@ -183,13 +191,115 @@ func extractConnections(args *cniArgs) error {
   return nil
 }
 
+func getResourcePrefix(args *cniArgs, resourceType string)(string,error){
+  confArgs, err := loadNetConf(args.stdIn)
```

Comment:
this is already done here :https://github.com/nokia/danm/blob/0ea0698fbd56252f8d303c58283b2e79c26f0a82/pkg/danm/danm.go#L103